### PR TITLE
feat: Enable Arc Gateway in AKSEE

### PR DESCRIPTION
**Pull request by whereiseva**

### CODE REVIEW

File: artifacts/Bootstrap.ps1 [hunk 1]
```diff
@@ -26,6 +28,8 @@ param (
 [System.Environment]::SetEnvironmentVariable('kubernetesDistribution', $kubernetesDistribution,[System.EnvironmentVariableTarget]::Machine)
 [System.Environment]::SetEnvironmentVariable('windowsNode', $windowsNode,[System.EnvironmentVariableTarget]::Machine)
 [System.Environment]::SetEnvironmentVariable('proxyCredentialsKeyVaultName', $proxyCredentialsKeyVaultName,[System.EnvironmentVariableTarget]::Machine)
+[System.Environment]::SetEnvironmentVariable('HELMREGISTRY', $helmRegistry,[System.EnvironmentVariableTarget]::Machine)
```
Comment (wuqiten) [inline artifacts/Bootstrap.ps1]: out of curiosity, what's this param use for?

File: artifacts/Bootstrap.ps1 [hunk 2]
```diff
@@ -11,7 +11,9 @@ param (
     [string]$windowsNode,
     [string]$kubernetesDistribution,
     [string]$uamiClientId,
-    [string]$proxyCredentialsKeyVaultName
+    [string]$proxyCredentialsKeyVaultName,
+    [string]$helmRegistry,
```
Comment (wuqiten) [inline artifacts/Bootstrap.ps1:15]: the SFF baremetal pipeline also using this script, but it does not populate the new parameters, can you test if this change will break the baremetal pipeline?

File: artifacts/Bootstrap.ps1 (hunk 1 above)
Comment (whereiseva) [reply artifacts/Bootstrap.ps1]: used to pull the arc agent. Arc Gateway team host the arc agent in a different repo.

**Review by haldekhelMSFT — APPROVED** [review]

File: artifacts/Bootstrap.ps1 (hunk 2 above)
Comment (whereiseva) [reply artifacts/Bootstrap.ps1:15]: added new parameters to baremetal pipeline in atlas repo

**Review by wuqiten — APPROVED** [review]

**Pull request merged**